src/ray/core_worker/transport/actor_scheduling_queue.h

```diff
@@ -97,6 +103,10 @@ class ActorSchedulingQueue : public SchedulingQueue {
   /// Whether we should enqueue requests into asyncio pool. Setting this to true
   /// will instantiate all tasks as fibers that can be yielded.
   bool is_asyncio_ = false;
+  /// Mutext to protect attributes used for thread safe APIs.
+  absl::Mutex mu_;
+  /// A map of actor task IDs -> is_canceled
+  absl::flat_hash_map<TaskID, bool> pending_tasks_queued_or_executing GUARDED_BY(mu_);
```

Comment:
I feel the name doesn't reflect the comment:

should we name it like `task_to_is_canceled`